src/erl.rs

```diff
@@ -1558,12 +1583,10 @@ impl TypeAst {
 
 impl Type {
     pub fn to_erlang_type_spec(&self) -> Document {
-        tracing::trace!("spec_for_type: {:?}", &self);
         match self {
             Self::Var { typ } => match &*Arc::as_ref(typ).borrow() {
-                TypeVar::Generic { .. } => "any()".to_doc(),
+                TypeVar::Generic { .. } | TypeVar::Unbound { .. } => "any()".to_doc(),
```

Comment:
Erlang's type specs support parametric polymorphism so we'll want to handle that. Adding it later would be a breaking change.

```erl
-type result(A, B) :: {ok, A} | {error, B}.
```

This is trickier as we'll need to have some context when rendering types so that the type variables are rendered correctly across all the types of a definition.

Reply:
Could you think of some example code that might cause this to emit the wrong type? Currently parametric types are working:
```rust
pub type A(a, b, c, d) {
  A(a)
  AB(a, b)
  DAB(d, a, b)
  BDACA(b, d, a, c, a)
}
```
erlang:
```
-type a(A, B, C, D) :: {a, A} |
    {ab, A, B} |
    {dab, D, A, B} |
    {bdaca, B, D, A, C, A}.
```

Reply:
Oh sorry, I have misread and misunderstood the code in that case. You're 2 steps ahead of me!